VirtoCommerce.Storefront/Startup.cs

```diff
@@ -163,12 +163,13 @@ public void ConfigureServices(IServiceCollection services)
             //Resource-based authorization that requires API permissions for some operations
             services.AddSingleton<IAuthorizationHandler, CanImpersonateAuthorizationHandler>();
             services.AddSingleton<IAuthorizationHandler, CanReadContentItemAuthorizationHandler>();
-            services.AddSingleton<IAuthorizationHandler, OnlyRegisteredUserAuthorizationHandler>();
+            services.AddSingleton<IAuthorizationHandler, OnlyRegisteredUserAuthorizationHandler>();            
```

Comment:
Whitespaces at the end of the line are not allowed.

Install https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.PowerCommandsforVisualStudio
Go to `Tools > Options > Productivity Power Tools > PowerCommands`
Enable `Format documents on save` and `Remove and Sort Usings on save`